membrane_curvature/tests/test_mdakit_membcurv.py

```diff
@@ -174,10 +268,23 @@ def universe():
 
 
 @pytest.fixture()
-def mdtraj_po4():
+def leaflets():
+    return ["lower", "upper"]
```

Comment:
This isn't *wrong*, but you could also just have the line below somewhere high up in the file, or pass in the leaflet labels with `pytest.mark.parametrize`.

```
LEAFLETS = ["lower", "upper"]
```

Fixtures are initialized and destroyed for each call of a test function (depending on the scope of the fixture) in a clean way to make sure that the input is always the same. That's not really necessary for a list of strings.